plugins/libctfcopytrace/ctfcopytrace.c

```diff
@@ -411,7 +411,7 @@ struct bt_ctf_stream_class *ctf_copy_stream_class(FILE *err,
 
 	type = bt_ctf_stream_class_get_event_header_type(stream_class);
 	if (type) {
-		if (override_ts64) {
+		if (override_ts64 && bt_ctf_trace_get_clock_class_count(writer_trace)) {
```

Comment:
Broke this down to check the `bt_ctf_trace_get_clock_class_count()` return value (which may be negative) separately.